python/graphscope/framework/graph.py

```diff
@@ -634,7 +645,7 @@ def add_edges(
             v_labels.append(dst_label)
 
         parent = self
-        if label in self.e_labels:
+        if self.evaluated != True and label in self.e_labels:
```

Comment:
```suggestion
        if not self.evaluated and label in self.e_labels:
```

Reply:
Thanks a lot for suggestion of elegant writing.